whois-nrtm4/src/main/java/net/ripe/db/nrtm4/JsonSerializer.java

```diff
@@ -0,0 +1,26 @@
+package net.ripe.db.nrtm4;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class JsonSerializer {
+
```

Comment:
Can we get rid of this class and use the object mapper directly as there is no logic here? Also we should get rid of  test class for this as well?

Reply:
done